scripts/testEnv.sh

```diff
@@ -129,9 +129,24 @@ export ORION_DATABASE_PASSWORD=${ORION_DATABASE_PASSWORD:-orion}
 # o CP4_DATABASE_NAME      - database for fourth context provider
 # o CP5_DATABASE_NAME      - database for fifth context provider
 #
-export CB_DATABASE_NAME=${CB_DATABASE_NAME:-ftest}
-export CP1_DATABASE_NAME=${CP1_DATABASE_NAME:-ftest_cp1}
-export CP2_DATABASE_NAME=${CP2_DATABASE_NAME:-ftest_cp2}
-export CP3_DATABASE_NAME=${CP3_DATABASE_NAME:-ftest_cp3}
-export CP4_DATABASE_NAME=${CP4_DATABASE_NAME:-ftest_cp4}
-export CP5_DATABASE_NAME=${CP5_DATABASE_NAME:-ftest_cp5}
+
+
+export DATABASE_NAME_BASE=CB
+export DATABASE_NAME_BASE_CP1=CP1
+export DATABASE_NAME_BASE_CP2=CP2
+export DATABASE_NAME_BASE_CP3=CP3
+export DATABASE_NAME_BASE_CP4=CP4
+export DATABASE_NAME_BASE_CP5=CP5
+export DATABASE_NAME_BASE_T1=t1
+export DATABASE_NAME_BASE_T2=t2
+export CB_DATABASE_NAME=${CB_DATABASE_NAME:-$DATABASE_NAME_BASE}
+export CP1_DATABASE_NAME=${CP1_DATABASE_NAME:-$DATABASE_NAME_BASE_CP1}
+export CP2_DATABASE_NAME=${CP2_DATABASE_NAME:-$DATABASE_NAME_BASE_CP2}
+export CP3_DATABASE_NAME=${CP3_DATABASE_NAME:-$DATABASE_NAME_BASE_CP3}
+export CP4_DATABASE_NAME=${CP4_DATABASE_NAME:-$DATABASE_NAME_BASE_CP4}
+export CP5_DATABASE_NAME=${CP5_DATABASE_NAME:-$DATABASE_NAME_BASE_CP5}
+export CBT1_DATABASE_NAME=${CBT1_DATABASE_NAME:-$DATABASE_NAME_BASE:-$DATABASE_NAME_BASE_T1}
+export CBT2_DATABASE_NAME=${CBT2_DATABASE_NAME:-$DATABASE_NAME_BASE:-$DATABASE_NAME_BASE_T2}
+export CP1T1_DATABASE_NAME=${CP1T1_DATABASE_NAME:-$DATABASE_NAME_BASE_CP1:-$DATABASE_NAME_BASE_T1}
```

Comment:
I think this is exagerrating a little.   "t1" is simply the name of a tenant.
We can't really create new env vars for each tenant name, we probably have 20 already in the functests and new tenant names can come in every day ...

The tenant name is the second parameter to the 'new dbInit/Drop', right?
Let's keep it as a simple text, freely selectable by the author of the functest.

*My Opinion, of course - more opinions are more than welcome*